VideoApp/VideoApp/Video/Participants/RemoteParticipant.swift

```diff
@@ -27,23 +27,22 @@ class RemoteParticipant: NSObject, Participant {
         
         return micTrack.isTrackSubscribed && micTrack.isTrackEnabled
     }
-    var isDominantSpeaker = false { didSet { postUpdate() } }
+    var isDominantSpeaker = false
     var isPinned = false
```

Comment:
It was a little strange to `postUpdate` from here. And also kind of wrong to not also be doing the same for `isPinned` although that was basically covered another way. Anyway some strange stuff I'm glad is gone.